cpu/atmega256rfr2/periph/pm.c

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
I think there is a reboot in the common folder which will be linked and used instead of this. 
Please make sure wich one is linked and if the common is used add softreset for that there and remove the reboot from this file. And make all neccessary chnage to the file there.